client/src/components/CommentCard.jsx

```diff
@@ -20,6 +22,18 @@ export default function CommentCard({ text, userName, date, rating }) {
         </div>
         <Card.Text>{text}</Card.Text>
         <div style={{ fontStyle: 'italic', fontSize: '0.85rem' }}>{date}</div>
+
+        {
+            icon &&
+            <div>
+                <AiFillEdit size={32} style={{color: 'red'}}/>
+                <MdDelete size={32} style={{color: 'green'}}/>
```

Comment:
It looks like your trash icon is green while your edit button is red. 

This might be confusing to users as people often associate the color green with adding and red with deleting instead

Reply:
Thank you for this reminder, you are right!